apix-integrationtests-client/alfresco/build.gradle

```diff
@@ -0,0 +1,85 @@
+def integrationTestsProjectDir = project.projectDir
+allprojects {
+    // Subproject of supported Alfresco versions
+    def subproject_alfresco_version = project.projectDir.name == "alfresco"
```

Comment:
It basically goes over "alfresco" , "231" and "232" folders and defaults the alfresco to the lowest supported version.

Reply:
Also, the Alfresco_version var is not available since this is needed to apply the source we need containing the versions. Hence this gradle trick.

The needed alfresco_version is imported from:     apply from: "${rootProject.projectDir}/alfresco/${subproject_alfresco_version}/overrides.gradle"

Reply:
This is basically a hack to set the Alfresco version and imports correct for the "alfresco" folder, since here the source code is placed. Moving the logix to subprojects makes us skip the " alfresco folder and only go over 231/232 folders. Breaking the code since no dependencies are available for the alfresco/src/code

Reply:
Here is how I fixed this before: https://github.com/xenit-eu/alfred-api/blob/25bdc63c77f9319db19b64d643b16f742d5efbc8/apix-impl/build.gradle#L8
IMO this is cleaner. However, this is already a big PR so ok to leave it in for now.